src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -52,6 +54,13 @@
     String getVersion();
 
     /**
+     * Returns the assigned logger to this {@link Plugin}.
+     *
+     * @return The assigned logger
+     */
+    Logger getLogger();
```

Comment:
@Zidane A plugin that wanted to troll server admins could grab the logger of another plugin and log a bunch of errors as if they were caused by another plugin. However, now that I think about it, they could just use `LoggerFactory` (see @Minecrell's comment).